etl/steps/export/multidim/education/latest/enrolment_rates.py

```diff
@@ -0,0 +1,266 @@
+
+    # Iterate and populate mappings
+    for var in cols_to_add_dimensions:
+        # --- Level ---
+        level = None
+        for key, val in level_keywords.items():
+            if key in var:
+                level = val
+                break
+        level_mapping[var] = level
+
+        # --- Enrolment Type ---
+        if "gross" in var:
+            enrolment_type_mapping[var] = "gross_enrolment"
+        elif "net" in var:
+            enrolment_type_mapping[var] = "net_enrolment"
+        elif "pre_enrr" in var:
+            enrolment_type_mapping[var] = "gross_enrolment"
+
+        # --- Sex ---
+        sex = None
+        for key, val in sex_keywords.items():
+            if f"__{key}__" in var or var.endswith(f"_{key}"):
+                sex = val
+                break
+        sex_mapping[var] = sex or "both"  # fallback for non-disaggregated vars
+
+    for col in tb.columns:
+        if col in ["country", "year"]:
+            continue
+        tb[col].metadata.original_short_name = "enrolment_rates"
+        tb[col].metadata.dimensions = {}
+
+        # Set dimensions
+        tb[col].metadata.dimensions["enrolment_type"] = enrolment_type_mapping[col]
+        tb[col].metadata.dimensions["level"] = level_mapping[col]
+        tb[col].metadata.dimensions["sex"] = sex_mapping[col]
+
+    # Add dimension definitions at table level
+    if isinstance(tb.metadata.dimensions, list):
+        tb.metadata.dimensions.extend(
+            [
+                {"name": "Metric", "slug": "enrolment_type"},
+                {"name": "Education level", "slug": "level"},
+                {"name": "Gender", "slug": "sex"},
+            ]
+        )
+
+    return tb
+
+
+def edit_indicator_displays(view):
+    """Edit display names for the grouped views."""
+    if view.dimensions["level"] == "level_side_by_side":
+        assert view.indicators.y is not None
+        for indicator in view.indicators.y:
+            if (
+                "enrolment_rate__primary" in indicator.catalogPath
+                or "enrolment_ratio__primary__male" in indicator.catalogPath
+            ):
+                indicator.display = {
+                    "name": "Primary",
+                }
+            elif "enrolment_rate__pre_primary" in indicator.catalogPath or "pre_enrr" in indicator.catalogPath:
+                indicator.display = {
+                    "name": "Pre-primary",
+                }
+            elif "lower_secondary" in indicator.catalogPath:
+                indicator.display = {
+                    "name": "Lower secondary",
+                }
+
+            elif "upper_secondary" in indicator.catalogPath:
+                indicator.display = {
+                    "name": "Upper secondary",
+                }
+            elif "tertiary" in indicator.catalogPath:
+                indicator.display = {
+                    "name": "Tertiary",
+                }
```

Comment:
I think you can improve the readability of this block by defining a temporary variable `display_name`:

```python
if (
    "enrolment_rate__primary" in indicator.catalogPath
    or "enrolment_ratio__primary__male" in indicator.catalogPath
):
    display_name = "Primary"
elif "enrolment_rate__pre_primary" in indicator.catalogPath or "pre_enrr" in indicator.catalogPath:
    display_name ="Pre-primary"
elif "lower_secondary" in indicator.catalogPath:
    display_name = "Lower secondary"
elif "upper_secondary" in indicator.catalogPath:
    display_name = "Upper secondary"
elif "tertiary" in indicator.catalogPath:
    display_name = "Tertiary"

indicator.display = {
    "name": display_name,
}
```